ceph_salt/config_shell.py

```diff
@@ -1114,6 +1114,8 @@ def check_config_prerequesites():
 
 def count_hosts(host_ls):
     all_nodes = PillarManager.get('ceph-salt:minions:all')
+    if all_nodes is None:
+        all_nodes = []
```

Comment:
if `PillarManager` were a `dict`, it could be simplified it to merely:

```
all_nodes = PillarManager.get('ceph-salt:minions:all', [])
```

but I don't know if `PillarManager` supports this syntax